pkg/addon/configpolicy/manifests/managedclusterchart/values.yaml

```diff
@@ -4,13 +4,10 @@ fullnameOverride: null
 nameOverride: null
 
 org: open-cluster-management
+replicas: 1
 
-enabled: true
 hubKubeconfigSecret: config-policy-controller-hub-kubeconfig
 
-clusterName: null
```

Comment:
I know clustername is set through the addonframework but still  I would leave these in the values.yaml as null.. just so all variables in the chart are listed/published  in values.yaml and can be identified by just looking at the values.yaml

Reply:
👍 It's just moved later in this file, to match the other charts' values.yaml format.